tests/check_if_exists_without_connect.phpt

```diff
@@ -8,6 +8,17 @@ if (!extension_loaded('zookeeper')) {
 --FILE--
 <?php
 $client = new Zookeeper();
-$client->exists('/test1');
+try {
```

Comment:
Please, see these docs bellow and be  a monster in making tests!

https://qa.php.net/write-test.php
https://qa.php.net/phpt_details.php